app/src/main/res/values/styles.xml

```diff
@@ -252,4 +252,56 @@
     <item name="fontPath">?defaultMediumTypeface</item>
   </style>
 
+  <style name="VenueInfo.Label" parent="None">
+    <item name="android:gravity">center_vertical</item>
```

Comment:
Not 100% necessary but, for clarity, put the horizontal gravity too: `start|center_vertical`